canvas_modules/common-canvas/src/common-canvas/svg-canvas-renderer.js

```diff
@@ -2073,11 +2089,11 @@ export default class SVGCanvasRenderer {
 	dragStart(d3Event, d) {
 		this.logger.logStartTimer("dragStart");
 		if (this.commentSizing) {
-			this.resizeObj = this.getComment(this.commentSizingId);
+			this.resizeObj = d;
 			this.initializeResizeVariables(this.resizeObj);
 
 		} else if (this.nodeSizing) {
-			this.resizeObj = this.getNode(this.nodeSizingId);
+			this.resizeObj = d;
```

Comment:
Looks like this `else` can be combined with the `if` above

Reply:
I had to undo the change mentioned above because I discovered the problem we were seeing with the original drag was caused by using d instead of getting the node/comment.